core/src/main/java/com/yahoo/ycsb/measurements/exporter/MeasurementsExporter.java

```diff
@@ -31,10 +31,10 @@
    * 
    * @param metric Metric name, for example "READ LATENCY".
    * @param measurement Measurement name, for example "Average latency".
-   * @param i Measurement to write.
+   * @param l Measurement to write.
    * @throws IOException if writing failed
    */
-  public void write(String metric, String measurement, int i) throws IOException;
```

Comment:
this change will be breaking. can we do this by adding an additional method with the long parameter and keep the one that's int?